pkg/pool-weighted/contracts/managed/ManagedPool.sol

```diff
@@ -190,23 +190,29 @@ contract ManagedPool is BaseWeightedPool, ProtocolFeeCache, ReentrancyGuard, ICo
         // Initialize the denormalized weight sum to ONE. This value can only be changed by adding or removing tokens.
         _denormWeightSum = FixedPoint.ONE;
 
+        bytes32 poolState;
```

Comment:
It feels like this is due some more ceremony: we're initializing all `poolState` fields, and should not forget any. We could at least list them here